.travis.yml

```diff
@@ -17,9 +17,9 @@ matrix:
   # to whatever the latest default is.
   include:
     - python: 2.7
-      env: "CRYPTO_BEFORE=1.6"
+      env: "CRYPTO_BEFORE=2.5"
     - python: 3.6
-      env: "CRYPTO_BEFORE=1.6"
+      env: "CRYPTO_BEFORE=2.5"
```

Comment:
I think you actually want 2.6 here - below it does:

```
pip install "cryptography<${CRYPTO_BEFORE}"
```

so 2.5 here will result in the latest 2.4.z being installed